docs/admin/memory.rst

```diff
@@ -49,6 +60,9 @@ Consider the following when determining the right value:
 - CrateDB uses `Memory mapped files`_ via `Lucene`_. This reduces the
   amount of heap space required and benefits from the file system cache.
 
+- Both to keep latency down, and to enable use of heap sizes above 32 GB,
+  it is also possible to use one of the newer garbage collectors, such as ZGC.
+  These are known to have worked, but aren't officially tested nor supported.
```

Comment:
I'd integrate this either in the first point that mentions G1GC and also include a link to GC configuration so users can learn how to change it. Or alternative put it into the `Recommendation` section further below under Memory Limits because it's kinda relevant to exceeding the 30.5 compressed oops space.